rita/Cargo.toml

```diff
@@ -16,35 +16,36 @@ default = []
 system_alloc = []
 
 [dependencies]
-clippy = { version = "*", optional = true }
```

Comment:
Does rustfmt start to sort deps now as well?

Reply:
No, it's just me, I forgot to split that to another commit, sorry. I divided this into a in-project dep pile and an external dep one.